### DISCUSSION TITLE
Waiting for a settings update that do not change anything

### DISCUSSION THREAD

**Discussion by qdequele** [meilisearch/product#591] [Feedback & Feature Proposal]
Hello @meilisearch/engine-team, I'm doing a demo on the IMDB database with v0.30.5. The database is 2.2M movie documents, and basically, for each movie, I have the title that has been translated into several languages. I would have instead per movies added all the translations in an array, but I wanted to show the localized title, and I wanted to create a big dataset. I'm trying to change my settings (mostly the ranking rules) to improve relevancy. Here are my database stats. 
<img width="367" alt="CleanShot 2023-01-22 at 11 24 46@2x" src="https://user-images.githubusercontent.com/6064892/213911215-d23999cf-9e20-4c9c-9c5c-6903d7809008.png">

The first thing I did was obviously to put all my text fields as searchable, text arrays, numbers, and boolean fields as filterable, and numbers as sortable. Then I tried multiple times to change the ranking rules. First, I added the ranking rule `rating:desc`, then I changed it to `date:desc`, then by `votes-count:desc`, and finally by `votes-count:desc, is-original-title:desc`. Each time it takes 5min to be able to test the new search. 5min is not a lot, but enough to go do something else and forget what I was doing. I created this issue because I think it's possible to have 0s update instead of 5min. It would improve the UX in general but mainly on the Cloud when we will have UI for configuring the indexes. 

<img width="367" alt="CleanShot 2023-01-22 at 11 21 52@2x" src="https://user-images.githubusercontent.com/6064892/213911267-f7a57746-25b3-4a58-904a-ef09f32919e4.png">

**Comment by qdequele** [comment]
Apparently, it's probably fixed in v1.0, I will have to test it :)

**Reply by qdequele** [reply]
We tested it with @Kerollmops this morning, and it is fixed in v1.0.0-rc.0!

**Reply by gmourier** [reply]
Thanks, @qdequele and @Kerollmops!
